testing/trino-product-tests/pom.xml

```diff
@@ -150,6 +150,16 @@
             <artifactId>java-driver-core</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
Why do we have the new dependencies as part of this PR?

Reply:
Product test `TestIcebergProcedureCalls.testMigrateHiveTableWithComplexType` we use the dependency to parse spark result.